plane/source/docs/additional-information.rst

```diff
@@ -0,0 +1,16 @@
+.. _additional-information:
+
+=============================================
+Additional Information and Topics of Interest
+=============================================
+
+
+.. toctree::
+    :maxdepth: 1
+
+    Upcoming Features <common-master-features>
+    Antenna Tracking <common-antenna-tracking>
+    Simulation <common-simulation>
+    OEM Customization <common-oem-customizations>
+    Use Cases and Applications<common-use-cases-and-applications>
```

Comment:
I think this needs a space

Reply:
not needed.....it is needed on https:/ links...ie  `this link <URL>`__